api/types/github.go

```diff
@@ -247,6 +256,38 @@ func (c *GithubConnectorV3) SetDisplay(display string) {
 	c.Spec.Display = display
 }
 
+// CheckExternalSSO returns an error if any of the Github organizations
+// specified in this connector use external SSO
+func (c *GithubConnectorV3) CheckExternalSSO() error {
+	for _, mapping := range c.GetTeamsToLogins() {
+		usesSSO, err := checkOrgExternalSSO(mapping.Organization)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		if usesSSO {
+			return trace.AccessDenied(
```

Comment:
nit: This error message is the same as the error message below. Can you extract the error/message to one place? You could probably extract the whole block inside `for` TBH.